snuba/datasets/storage.py

```diff
@@ -159,6 +159,7 @@ def __init__(
             mandatory_condition_checkers,
         )
         assert isinstance(schema, WritableTableSchema)
+
```

Comment:
nit: did `flake8` force these changes? I'd rather not commit only whitespace changes in a file